src/button/button.scss

```diff
@@ -28,44 +29,45 @@ $iui-button-padding-large: $iui-xs * 6;
   white-space: nowrap;
   border: 1px solid transparent;
   @media (prefers-reduced-motion: no-preference) {
-    transition: color $iui-speed-fast ease-out,
-      background-color $iui-speed-fast ease-out,
+    transition: color $iui-speed-fast ease-out, background-color $iui-speed-fast ease-out,
       border-color $iui-speed-fast ease-out;
   }
 
-  > .iui-icon:not(.iui-user-icon) {
-    width: $iui-icons-default;
-    height: $iui-icons-default;
-    @media (prefers-reduced-motion: no-preference) {
-      transition: fill $iui-speed-fast ease-out;
+  > .iui-button-icon {
+    @include themed {
+      fill: t(iui-icons-color-actionable);
     }
   }
 
-  &:enabled:hover {
+  &:hover {
     text-decoration: none;
   }
 
   @include iui-focus;
   @include iui-button-sizes;
+
+  &[disabled] {
+    @include iui-button-disabled;
+  }
 }
 
 @mixin iui-button-sizes {
```

Comment:
I think it would be nice to refactor this mixin to not require classes.

e.g. you could add `@include iui-button-sizes(large)` to header-button, and then you can use it like this:
```diff
- <button class="iui-button iui-large iui-header-button" ... />
+ <button class="iui-button iui-header-button" ... />
```

And then we can reduce the specificity of overrides done in header.scss.

Reply:
Changed this mixin

Reply:
Then we should improve some selectors in header.scss